### PR TITLE
Add_RO_country_fiche_2024

### PR DESCRIPTION
**Pull request by LauraBolocan**

### CODE REVIEW

**Comment by MarcoMinghini** [comment]
Dear @LauraBolocan,
we see you made two pull requests, but it seems that the work is still in progress. Could you confirm? If so, please consider making one single pull request with all the edits.
Thank you, and feel free to come back to us if you have questions.

**Comment by LauraBolocan** [comment]
Hello @MarcoMinghini 

I have deleted the other pull request and I've updated this one as the final Country Fiche. Could you tell me if it is ok?

Thank you

**Review by fabiovinci — APPROVED** [review]

File: RO/INIS_structure.PNG
[the remark this answers is not in the document]
Comment (fabiovinci) [reply RO/INIS_structure.PNG:1]: I removed the duplicated image.

File: RO/Country_fiches/INIS_structure.PNG
[the remark this answers is not in the document]
Comment (fabiovinci) [reply RO/Country_fiches/INIS_structure.PNG:1]: I created a dedicated folder for all the images and moved the existing one.

**Comment by fabiovinci** [comment]
Dear @LauraBolocan

I fixed some typos, updated some links and created a dedicated folder for the images.
Everything seems ok, I will proceed with the merge.
If there's anything else that needs to be adjusted, please don't hesitate to let us know.

**Pull request merged**